lib/components/Card/index.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import styled from 'styled-components';
+import theme from '../theme';
+
+import CardTitleGraphic from './CardTitleGraphic.png';
+
+const CardStyled = styled.div`
+  background: ${theme.colors.white};
+  color: ${theme.colors.text};
+
+  border-radius: ${theme.units.borderRadius};
+
+  /* So children don't flow outside border-radius. */
+  overflow: hidden;
+
+  /* So last child's bottom left/right borders don't get cut off. */
+  > :last-child {
+    border-bottom-right-radius: ${theme.units.borderRadius};
+    border-bottom-left-radius: ${theme.units.borderRadius};
+  }
+`;
+
+const CardTitleStyled = styled.div`
+  background: ${theme.colors.primary};
+  color: ${theme.colors.white};
+
+  font-weight: bold;
+  font-size: 18px;
+
+  border: 1px solid ${theme.colors.primary};
+
+  padding: ${theme.units.padding};
+
+  background-image: url(${CardTitleGraphic});
+  background-size: 167px 96px;
+  background-repeat: no-repeat;
+  background-position: top right -50px;
+`;
+
+const CardContentStyled = styled.div`
+  padding: ${theme.units.padding};
+
+  border-bottom: 1px solid ${theme.colors.grayLight};
+  border-left: 1px solid ${theme.colors.grayLight};
+  border-right: 1px solid ${theme.colors.grayLight};
+
+  /*
+    Setting margin-top and margin-bottom so that the spacing set by the above
+    padding is consistent regardless of the first/last element used.
+  */
+  > :first-child {
+    margin-top: 0;
+  }
+
+  > :last-child {
+    margin-bottom: 0;
+  }
```

Comment:
question: this brings up the question of "ownership" over style rules. This container is reaching down into its children and changing their margin.

1. Maybe this is good? Maybe margin specifically should have a rule like "children don't set their own margin"?
2. Maybe this is bad? Maybe the rule should be "components own their own styles"?
3. Maybe exceptions are inevitable and it's a rabbit-hole to try to make a rule for this?

In general I think we should state an ideal pattern so we're aware of when we're using it and when we're breaking it. I'll propose a really simple one to begin with:

> No descendant selectors.

That means setting things like `align-items` in flex container is ok. It affects children, but it's a property of the container. But things like `> :first-child` are not okay, because they're selecting descendants and modifying _their_ properties "at a distance" in ways that are hard to guess when you're looking at the child's component.

If we adopt that rule, you could solve this padding/margin problem by (1) not applying margin on children and (2) using a `space-between` flex container.

This is a complicated proposal, so I don't think it should hold up this PR, but I'm interested in your thoughts.